numpy/random/pcg64.pyx

```diff
@@ -131,17 +131,17 @@ cdef class PCG64(BitGenerator):
 
         Notes
         -----
-        The step size is phi-1 when divided by 2**128 where phi is the
-        golden number.
+        The step size is phi-1 when multiplied by 2**128 where phi is the
+        golden ratio.
         """
-        step = 0x9e3779b97f4a7c15f39cc0605cedc834
+        step = 0x9e3779b97f4a7c15f39cc0605cedc835
```

Comment:
I need a LGTM on this one last non-trivial bit, then we can merge this. @bashtage ?

Reply:
Feel free to round it which ever way that you prefer. LGTM.

Reply:
Thank you for your indulgence. :-)